src/protocol.cpp

```diff
@@ -51,6 +51,19 @@ MAIN FRAME
 
 
 
+SPLIT MESSAGE CONTAINER
+-----------------------
+
+    +------------+------------------------+------------------+--------------+
+    | 2 bytes ID | 1 byte number of parts | 1 byte split cnt | n bytes data |
+    +------------+------------------------+------------------+--------------+
+
+- ID is the message ID of the message being split
```

Comment:
`number of parts` - total number of parts comprising the whole message

Reply:
done